superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -6,6 +6,7 @@
 import os
 
 from superbench.common.utils import logger
+from superbench.common.utils.topo_aware import gen_topo_aware_config
```

Comment:
export it in "superbench/common/utils/__init__.py" and import from "superbench.common.utils" directly?